templates/federated_reporting/nova_api.py

```diff
@@ -0,0 +1,134 @@
+#!/var/cfengine/bin/cfengine-selected-python
```

Comment:
Either no shebang or this should do something as a script.

Reply:
sure.